sandy/sections/mf2.py

```diff
@@ -182,272 +186,333 @@ def read_mf2(tape, mat):
-                    LRU2_LFW0_LRF1 ={}
+            elif LRU == 2:
+                if LFW == 0 and LRF == 1:
+                    LRU2_LFW0_LRF1 = {}
                     LRU2_LFW0_LRF1.update(header2)
                     C, i = sandy.read_cont(df, i)
                     add = {
-                            "SPI": C.C1,
-                            "AP": C.C2,
-                            "LSSF": C.L1,                   #Flag governing the interpretation of the File 3 cross sections
-                            }
+                        "SPI": C.C1,
+                        "AP": C.C2,
+                        "LSSF": C.L1,  # Flag governing the interpretation of the File 3 cross sections
+                    }
                     NLS = int(C.N1)
                     LRU2_LFW0_LRF1.update(add)
                     LRU2_LFW0_LRF1_NLS = {}
                     for k in range(NLS):
                         L, i = sandy.read_list(df, i)
                         add = {
-                                "AWRI": L.C1,
-                                "L": L.L1,
-                                "NJS": L.N2,                    #Number of J-states for a particular l-state
-                            }
-                        keys = ["D", "AJ", "AMUN", "GN0", "GG","0"]
-                        RES_PAR = [dict(zip(keys, items)) for items in sandy.utils.grouper(L.B, 6)]
-                        add.update({"RES_PAR":RES_PAR})
-                        LRU2_LFW0_LRF1_NLS.update({k : add})
+                            "AWRI": L.C1,
+                            "L": L.L1,
+                            "NJS": L.N2,  # Number of J-states for a particular l-state
+                        }
+                        keys = ["D", "AJ", "AMUN", "GN0", "GG", ]
+                        RES_PAR = [dict(zip(keys, items))
+                                   for items in sandy.utils.grouper(L.B, 6)]
+                        add.update({"RES_PAR": RES_PAR})
+                        LRU2_LFW0_LRF1_NLS.update({k: add})
                     LRU2_LFW0_LRF1.update({"L": LRU2_LFW0_LRF1_NLS})
                     dico[(EL, EH)] = LRU2_LFW0_LRF1
-                elif LFW == 1 and LRF == 1 :
-                    raise ValueError("LFW = 1  LRF = 1 is not supported in SANDY")
-                elif LRF == 2 : 
+                elif LFW == 1 and LRF == 1:
+                    raise ValueError(
+                        "LFW = 1  LRF = 1 is not supported in SANDY")
+                elif LRF == 2:
                     LRU2_LRF2 = {}
                     LRU2_LRF2.update(header2)
                     C, i = sandy.read_cont(df, i)
                     add = {
-                            "SPI": C.C1,
-                            "AP": C.C2,
-                            "LSSF": C.L1,
-                            }
-                    NLS = int (C.N1)
+                        "SPI": C.C1,
+                        "AP": C.C2,
+                        "LSSF": C.L1,
```

Comment:
Description needed

Reply:
it is already described above in LRU = 2 LRF = 1